benches/benches/vm_set/alu.rs

```diff
@@ -37,7 +37,8 @@ pub fn run(c: &mut Criterion) {
     run_group_ref(
         &mut c.benchmark_group("aloc"),
         "aloc",
-        VmBench::new(op::aloc(0x10)),
+        VmBench::new(op::aloc(0x10))
+            .with_prepare_script(vec![op::movi(0x10, (1 << 18) - 1)]),
```

Comment:
The sanity check we are doing `0` and here we do `(1 << 18) - 1`. Is that okay that they are different?

Reply:
With a sanity check, we may run out of memory before running out of gas. While in cost benchmarks we revert the state of VM after each iteration.

There is no difference in using `0` or `u18::MAX` with the current implementation, but we need this difference to potentially catch mismatching in sanity check benchmarks and cost benchmarks.